Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM yastdevel/cpp
+RUN zypper --gpg-auto-import-keys --non-interactive in --no-recommends \
+  "pkgconfig(gio-unix-2.0)" \
+  "pkgconfig(glib-2.0) >= 2.31.0" \
+  "pkgconfig(gtk+-3.0)" \
+  "pkgconfig(libgnome-menu-3.0)"
```

Comment:
Exactly, we have two Docker images (for C++ YaST packages and Ruby YaST packages), both install a common set of packages.

You need to find the right balance between a big image with all packages (takes longer time at Travis to download) and a small image (resulting in extra package downloads at Travis).

These packages have big dependencies and are required only for this very specific package so they are not part of the base image and are installed additionaly here.